client/src/modules/list/model/actionTypes.js

```diff
@@ -17,6 +17,7 @@ export const ListActionTypes = Object.freeze({
   FETCH_META_DATA_FAILURE: 'list/FETCH_META_DATA_FAILURE',
   FETCH_META_DATA_REQUEST: 'list/FETCH_META_DATA_REQUEST',
   FETCH_META_DATA_SUCCESS: 'list/FETCH_META_DATA_SUCCESS',
+  REMOVE_ARCHIVED_META_DATA: 'list/REMOVE_ARCHIVED_META_DATA',
```

Comment:
should here be request success and failure also?

Reply:
Archived lists are removing from Redux only. We don't send a request to the server in this case. So I think there is no need to create failure, success and request actions.